Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py

```diff
@@ -76,11 +80,50 @@ def get_event_status(self, job_id, event_id):
             raise Exception(f"Error HttpCode={raw_response.status_code}")
         return raw_response.json()
 
+    def send_feedback(self, job_id, event_id, username, arcanna_label, closing_notes, indicators):
+        url_suffix = f"api/v1/events/{job_id}/{event_id}/feedback"
+        body = self.map_to_arcanna_label(arcanna_label, closing_notes, username)
+        demisto.log(f"{body}")
+        if indicators is not None:
+            body["indicators"] = indicators
+        raw_response = requests.put(url=self.base_url + url_suffix, headers=self.get_headers(), verify=self.verify,
+                                    json=body)
+
+        if raw_response.status_code != 200:
+            raise Exception(f"Error HttpCode={raw_response.status_code}")
```

Comment:
Don't you want to give the uses a bit more info about the error? 
```suggestion
            raise Exception(f"Arcanna Error HttpCode={raw_response.status_code}")
```

Reply:
I think it's perfectly fine as it is